wallet/src/main/scala/org/bitcoins/wallet/models/DLCAcceptTable.scala

```diff
@@ -2,22 +2,30 @@ package org.bitcoins.wallet.models
 
 import org.bitcoins.core.crypto._
 import org.bitcoins.core.currency.CurrencyUnit
-import org.bitcoins.core.protocol.Bech32Address
+import org.bitcoins.core.protocol.BitcoinAddress
 import org.bitcoins.core.protocol.transaction.OutputReference
+import org.bitcoins.core.psbt.InputPSBTRecord.PartialSignature
 import org.bitcoins.dlc.DLCMessage.DLCAccept
 import org.bitcoins.dlc.{CETSignatures, DLCPublicKeys}
 import slick.jdbc.SQLiteProfile.api._
-import slick.lifted.{PrimaryKey, ProvenShape}
+import slick.lifted.{ForeignKeyQuery, PrimaryKey, ProvenShape}
 
 case class DLCAcceptDb(
```

Comment:
Why is the auto-incrementing primary id not represented here?

Reply:
None of the other tables that have an auto-incrementing primary id have it in their `XDb` or anywhere in the `XTable.scala` at all.

Reply:
I think i'm on the record saying I would like autoinc primary key, so we can decouple the protocol itself from our own internal representation. Plus we have a a bunch of infra to get this stuff for free `DbRowAutoInc`, `CRUDAutoInc`, `TableAutoInc` etc

Reply:
I think you might have misunderstood my intention here, I'm saying that since the auto inc primary key is in the db, it should be a parameter in these `case classes`

Reply:
Also just generally it seems that our slick tables only represent a subset of the real ones

Reply:
(If I use slick to generate this table it will not have an autoinc primary key)